src/utils/device/dehydration.ts

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { logger } from "matrix-js-sdk/src/logger";
+
+import { MatrixClientPeg } from "../../MatrixClientPeg";
+
+// the interval between creating dehydrated devices
+const DEHYDRATION_INTERVAL = 7 * 24 * 60 * 60 * 1000;
+
+// check if device dehydration is enabled
```

Comment:
Please can you make these comments into proper doc comments.

It wouldn't hurt to give a bit more information on what exactly "enabled" means, too.